shopfloor/services/cluster_picking.py

```diff
@@ -324,6 +324,7 @@ def _sort_key_lines(line):
             line.location_id.shopfloor_picking_sequence or "",
             line.location_id.name,
             -int(line.move_id.priority or 1),
+            line.move_id.date_expected or "",
```

Comment:
`date_expected` being a datetime, we will have a comparison error if one is empty (can it?)

Reply:
AFAIS is always valued. I could add a safer default though.

Reply:
```py
date_expected = fields.Datetime(
        'Expected Date', default=fields.Datetime.now, index=True, required=True,
        states={'done': [('readonly', True)]},
        help="Scheduled date for the processing of this move")
```
I can even drop the `or ""` IMO.

Reply:
> I can even drop the or "" IMO.

:+1:

Reply:
done